### PR TITLE
Frontend (dom, css, hbs) edits for the donate

### PR DESCRIPTION
**Pull request by AbdallahAmmar96**
edit the style #91  #63  #34 
edit the dom and add the validation #61  #91  #34 
add h2 in the hbs file for show the user message if something went wrong #34 
add the method post in the controllers 
delete file from main folder and adit to the user folder

### CODE REVIEW

**Review by InassTubail — CHANGES_REQUESTED** [review]
unify your querySelector method in "public/js/user/donateDom.js"  as your team members @aminalakhsham in this file
[querySelector method](https://github.com/FACG6/enhance-a-smile/pull/89#discussion_r267047256)

**Review by InassTubail — CHANGES_REQUESTED** [review]
unify your method of writing CSS classes name as your team members.

File: public/js/user/donateDom.js [hunk 1]
```diff
@@ -8,35 +8,61 @@ const popUpSection = document.querySelector('.popUpBack');
...
+        'Content-Type': 'application/json',
+      },
+    })
+      .then(result => result.json())
+      .then((serverRes) => {
+        if (serverRes.msg === 'donation added sucsesfully') {
+          popUpSection.classList.remove('hide');
+        }
+      })
+      .catch(err => console.log(err));
```
Comment (InassTubail) [inline public/js/user/donateDom.js:129]: Remove console.log

**Review by InassTubail — APPROVED** [review]

**Comment by AbdallahAmmar96** [comment]
there is one console.log will be removed after adding the database

**Comment by AbdallahAmmar96** [comment]
edit the style color in the taps when click on the top tap

File: public/js/user/donateDom.js [hunk 2]
```diff
@@ -1,42 +1,133 @@
...
+    fullName,
+    phoneNumber,
+    cityName,
+    email,
+  } = personalInformation;
+  if (!fullName) {
+    valditMsg.textContent = 'Please enter your full name';
+  } else if (!phoneNumber) {
+    valditMsg.textContent = 'Please enter your phone number';
+  } else if ((!/^[0-9]{10}$/.test(phoneNumber))) {
```
Comment (tomduggan85) [inline public/js/user/donateDom.js:57]: @AbdallahAmmar96 Hey I had a comment in another PR about phone number regexes that is valid here too:
https://github.com/FACG6/enhance-a-smile/pull/94#discussion_r267339537

Comment (AbdallahAmmar96) [reply public/js/user/donateDom.js:57]: ok I'll consider it

**Pull request merged**